### PR TITLE
Add com.github.elth0r0.iqpuzzle

### PR DESCRIPTION
**Pull request by ElTh0r0**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project.
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by ElTh0r0** [comment]
bot, build com.github.elth0r0.iqpuzzle

File: com.github.elth0r0.iqpuzzle.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+app-id: com.github.elth0r0.iqpuzzle
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: iqpuzzle
+separate-locales: false
+rename-icon: iqpuzzle
+finish-args:
+  - --share=ipc
+  - --socket=x11
```
Comment (nedrichards) [inline com.github.elth0r0.iqpuzzle.yml:10]: you don't need this and fallback-x11. just one or the other (fallback-x11 means wayland will be chosen by default if available)

**Comment by nedrichards** [comment]
bot, build com.github.elth0r0.iqpuzzle

File: com.github.elth0r0.iqpuzzle.yml (hunk 1 above)
Comment (bilelmoussaoui) [reply com.github.elth0r0.iqpuzzle.yml:10]: You should have instead removed socket=x11 and not fallback-x11.

Comment (ElTh0r0) [reply com.github.elth0r0.iqpuzzle.yml]: Many thanks for the information! The explanation in the Flatpak docs wasn't 100% clear to me, which combination should be used. Now I exchanged x11 with fallback-x11.

**Comment by ElTh0r0** [comment]
bot, build com.github.elth0r0.iqpuzzle

**Pull request closed without merging**